services/visual-studio-app-center/visual-studio-app-center-builds.service.js

```diff
@@ -0,0 +1,62 @@
+'use strict'
+
+const Joi = require('@hapi/joi')
+const { isBuildStatus, renderBuildStatusBadge } = require('../build-status')
+const {
+  BaseVisualStudioAppCenterService,
+  keywords,
+  documentation,
+} = require('./visual-studio-app-center-base')
+const { NotFound } = require('..')
+
+const schema = Joi.array().items({
+  result: isBuildStatus.required(),
+})
+
+module.exports = class VisualStudioAppCenterBuilds extends BaseVisualStudioAppCenterService {
+  static get category() {
+    return 'build'
+  }
+
+  static get route() {
+    return {
+      base: 'vsac/builds',
+      pattern: ':owner/:app/:branch/:token',
```

Comment:
Are both `branch` and `token` required values here? I'm not familiar with this service but it seems with other build services that `branch` is optional, and if so we'd want to have the route params reflect that with an optional branch param

Reply:
Token is always required, but builds is the only endpoint that requires (or can even use) the branch variable.

Releases just use sequential IDs.

Reply:
So a branch has to be specified explicitly? The API doesn't use a default branch if none is specified?

Reply:
No, it requires a branch to be specified.